### PR TITLE
i18n(fr): Add `reference/experimental-flags/*`

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Add `reference/experimental-flags/*`

- [i18n(fr):Add reference/experimental-flags/svg.mdx](https://github.com/withastro/docs/commit/d76f6729e5d48dd0d077dfbed1758c36c0736fd4) 
- [i18n(fr):Add reference/experimental-flags/sessions.mdx](https://github.com/withastro/docs/commit/905fda878613c3bb4d6f04a42572329c40409850) 
- [i18n(fr):Add reference/experimental-flags/responsive-images.mdx](https://github.com/withastro/docs/commit/4fe4e8f0f0da72b8203b9b16eb3b1b695ced50ba) 
- [i18n(fr):Add reference/experimental-flags/index.mdx](https://github.com/withastro/docs/commit/91c4b6902554ee91e1aa05f72fe3af146f3f5dd2) 
- [i18n(fr):Add reference/experimental-flags/content-intellisense.mdx](https://github.com/withastro/docs/commit/3b0054751af5bad246fb000d145eda8007937963) 
- [i18n(fr):Add reference/experimental-flags/client-prerender.mdx](https://github.com/withastro/docs/commit/9af202a53eeafffedacb4a9bbc57dd5d38a9646f)

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/reference/experimental-flags/content-intellisense.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+```js
+{
+  experimental: {
+    contentIntellisense: true,
+  },
+}
+```
+
+Pour utiliser cette fonctionnalité avec l'extension Astro VS Code, vous devez également activer l'option `astro.content-intellisense` dans les paramètres de VS Code. Pour les éditeurs utilisant directement le serveur de langue Astro, passez le paramètre d'initialisation `contentIntellisense : true` pour activer cette fonctionnalité.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/reference/experimental-flags/content-intellisense.mdx:29]: no space:
```suggestion
Pour utiliser cette fonctionnalité avec l'extension Astro VS Code, vous devez également activer l'option `astro.content-intellisense` dans les paramètres de VS Code. Pour les éditeurs utilisant directement le serveur de langue Astro, passez le paramètre d'initialisation `contentIntellisense: true` pour activer cette fonctionnalité.
```

File: src/content/docs/fr/reference/experimental-flags/index.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    },
+});
+```
+
+
+Astro propose des flags expérimentaux afin de donner aux utilisateurs un accès anticipé aux nouvelles fonctionnalités pour qu'ils les testent et fassent part de leurs commentaires.
+
+Ces flags vous permettent de participer au développement des fonctionnalités en signalant les problèmes et en partageant vos opinions. La stabilité de ces fonctionnalités n'est pas garantie et elles peuvent inclure des ruptures, même dans de petites versions de `patch`, alors que la fonctionnalité est activement développée.
+
+Nous recommandons de [mettre à jour Astro](/fr/upgrade-astro/#mise-à-jour-vers-la-dernière-version) fréquemment, et de suivre les notes de version dans le [Astro changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) qui vous informera de tout changement nécessaire au code de votre projet. La documentation sur les fonctionnalités expérimentales sera toujours mise à jour pour la version actuelle uniquement.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/reference/experimental-flags/index.mdx:24]: Either `changelog d'Astro` or `journal des modifications d'Astro` but not `Astro changelog`:
```suggestion
Nous recommandons de [mettre à jour Astro](/fr/upgrade-astro/#mise-à-jour-vers-la-dernière-version) fréquemment, et de suivre les notes de version dans le [changelog d'Astro](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) qui vous informera de tout changement nécessaire au code de votre projet. La documentation sur les fonctionnalités expérimentales sera toujours mise à jour pour la version actuelle uniquement.
```

File: src/content/docs/fr/reference/experimental-flags/responsive-images.mdx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  width="800"
+  height="600"
+  style="--w: 800; --h: 600; --fit: cover; --pos: center;"
+  data-astro-image="responsive"
+>
+```
+
+Le style suivant est appliqué pour garantir que les images se redimensionnent correctement :
+
+```css title="Responsive Image Styles"
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/reference/experimental-flags/responsive-images.mdx:66]:
```suggestion
```css title="Styles des images réactives"
```

File: src/content/docs/fr/reference/experimental-flags/sessions.mdx [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+```
+
+Les sessions s'appuient sur un [pilote de session configurable](#activation-des-sessions-expérimentales) pour stocker les données sur l'objet `session`. Un [cookie de session](#cookies) stocke un identifiant de session.
+
+L'objet [`session`](#api-sessions) vous permet d'interagir avec l'état de l'utilisateur stocké (par exemple, ajouter des articles à un panier d'achat) et l'identifiant de session (par exemple, supprimer le cookie d'identifiant de session lors de la déconnexion).
+
+
+## Activation des sessions expérimentales
+
+Dans cette première version de `experimental.sessions`, les adaptateurs de serveur officiels d'Astro n'ont pas encore été mis à jour pour fournir un pilote de session par défaut basé sur la fonctionnalité de stockage par défaut de la plate-forme. Jusqu'à ce qu'ils soient intégrés, vous devrez spécifier vous-même un `driver` dans l'objet de configuration `experimental.sessions` en plus d'[ajouter un adaptateur](/fr/guides/on-demand-rendering/#dd-an-adapter).
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/reference/experimental-flags/sessions.mdx:39]: Fix link:
```suggestion
Dans cette première version de `experimental.sessions`, les adaptateurs de serveur officiels d'Astro n'ont pas encore été mis à jour pour fournir un pilote de session par défaut basé sur la fonctionnalité de stockage par défaut de la plate-forme. Jusqu'à ce qu'ils soient intégrés, vous devrez spécifier vous-même un `driver` dans l'objet de configuration `experimental.sessions` en plus d'[ajouter un adaptateur](/fr/guides/on-demand-rendering/#add-an-adapter).
```

File: src/content/docs/fr/reference/experimental-flags/svg.mdx [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+  label: Composants SVG
+i18nReady: true
+---
+
+import Since from '~/components/Since.astro'
+
+<p>
+
+**Type :** `boolean`<br />
+**Défaut :** `false`<br />
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/reference/experimental-flags/svg.mdx:13]:
```suggestion
**Par défaut :** `false`<br />
```

File: src/content/docs/fr/reference/experimental-flags/svg.mdx [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+<Logo size={64} mode="sprite" />
+```
+
+
+## experimental.svg.mode
+
+<p>
+
+**Type :** `string`<br />
+**Défaut :** `'inline'`<br />
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/reference/experimental-flags/svg.mdx:82]:
```suggestion
**Par défaut :** `'inline'`<br />
```

File: src/content/docs/fr/reference/experimental-flags/svg.mdx [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+      mode: 'sprite',
+    }
+  },
+}
+```
+
+- `inline` : Astro va intégrer le contenu SVG dans votre sortie HTML. (par défaut)
+- `sprite` : Astro va générer une feuille de sprite avec tous les fichiers SVG importés.
+
+Pour une présentation complète et pour donner votre avis sur cette API expérimentale, consultez le [SVG feature RFC](https://github.com/withastro/roadmap/pull/1035).
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/reference/experimental-flags/svg.mdx:101]:
```suggestion
Pour une présentation complète et pour donner votre avis sur cette API expérimentale, consultez la [RFC de la fonctionnalité SVG](https://github.com/withastro/roadmap/pull/1035).
```

**Review by ArmandPhilippot — CHANGES_REQUESTED** [review]
This should fix the broken link and an inline code. The rest is mostly nitpicking.

And as discussed in #10467, I still think we should translate `flags` in these files... but maybe we need more opinions on this. At worst, we can still merge and make the translation more consistent later...

**Review by ArmandPhilippot — APPROVED** [review]
Well, LGTM! But still an issue with `astro check`... 👀

File: src/content/docs/fr/reference/experimental-flags/index.mdx [hunk 8]
```diff
@@ -0,0 +1,24 @@
+---
+title: Configuration des options expérimentales
+i18nReady: true
+---
+
+Les fonctionnalités expérimentales ne sont disponibles qu'après l'activation d'un flag dans le fichier de configuration d'Astro.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/reference/experimental-flags/index.mdx:6]:
```suggestion
Les fonctionnalités expérimentales ne sont disponibles qu'après l'activation d'une option dans le fichier de configuration d'Astro.
```

File: src/content/docs/fr/reference/experimental-flags/index.mdx [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+export default defineConfig({
+    experimental: {
+        // activer les options expérimentales
+        // pour tester de nouvelles fonctionnalités
+    },
+});
+```
+
+
+Astro propose des flags expérimentaux afin de donner aux utilisateurs un accès anticipé aux nouvelles fonctionnalités pour qu'ils les testent et fassent part de leurs commentaires.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/reference/experimental-flags/index.mdx:20]:
```suggestion
Astro propose des options expérimentales afin de donner aux utilisateurs un accès anticipé aux nouvelles fonctionnalités pour qu'ils les testent et fassent part de leurs commentaires.
```

File: src/content/docs/fr/reference/experimental-flags/index.mdx [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+Ces flags vous permettent de participer au développement des fonctionnalités en signalant les problèmes et en partageant vos opinions. La stabilité de ces fonctionnalités n'est pas garantie et elles peuvent inclure des ruptures, même dans de petites versions de `patch`, alors que la fonctionnalité est activement développée.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/reference/experimental-flags/index.mdx:22]:
```suggestion
Ces options vous permettent de participer au développement des fonctionnalités en signalant les problèmes et en partageant vos opinions. La stabilité de ces fonctionnalités n'est pas garantie et elles peuvent inclure des ruptures, même dans de petites versions de `patch`, alors que la fonctionnalité est activement développée.
```

**Review by ArmandPhilippot — COMMENTED** [review]
You missed 3 flags. 😉

**Comment by thomasbnt** [comment]
> You missed 3 flags. 😉

I was editing it. Thanks

**Pull request merged**